DigitalLearningSolutions.Web/Views/TrackingSystem/CentreConfiguration/Index.cshtml

```diff
@@ -0,0 +1,27 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem
+@model CentreConfigurationViewModel
+@{
+  ViewData["Title"] = "Centre Configuration";
+}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-full">
+    <h1 id="page-heading" class="nhsuk-heading-xl">Centre Configuration</h1>
+    <div class="nhsuk-card">
+      <div class="nhsuk-card__content">
+        <table class="nhsuk-table">
```

Comment:
I think we should be using a <dl class="nhsuk-summary-list"> here rather than a table, the same as the MyAccount Index

Reply:
https://service-manual.nhs.uk/design-system/components/summary-list